gym/envs/classic_control/acrobot.py

```diff
@@ -267,6 +265,9 @@ def _dsdt(self, s_augmented):
         return (dtheta1, dtheta2, ddtheta1, ddtheta2, 0.0)
 
     def render(self, mode="human"):
+        import pygame
```

Comment:
I think we could instead shift the imports into the if clause: `if self.screen is None:` so we don't need to call the import statement every time we call `render()`

Reply:
The same comment about static code checkers applies here. The repeated import shouldn't be a performance problem fortunately, python is smart enough to avoid actually doing the whole import again.